src/apps/watchfaces/watchface_dual_time.cpp

```diff
@@ -0,0 +1,294 @@
+#include "./apps/watchfaces/watchface_dual_time.h"
+
+#include <config.h>
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_config_keys.h>
+#include <osw_hal.h>
+#include <time.h>
+
+#include "./apps/watchfaces/watchface.h"
+#include "bma400_defs.h"
+
+#define COLOR_BLACK rgb565(0, 0, 0)
+
+#define BUF_W 240
```

Comment:
This is the display size, right? We already have a `define` for that - please use it instead.

Reply:
Got it, `DISP_W`, `DISP_H`..10x